onnx/defs/experiments/defs.cc

```diff
@@ -29,7 +29,7 @@ is applied to the tensor elementwise.
     .Attr("beta" , "Value of beta", AttributeProto::FLOAT, OPTIONAL)
     .Input(0, "X", "1D input tensor", "T")
     .Output(0, "Y", "1D output tensor", "T")
-    .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
+    .TypeConstraint("T", { TensorType<TensorProto::FLOAT16>::Type(), TensorType<TensorProto::FLOAT>::Type(), TensorType<TensorProto::DOUBLE>::Type() },
```

Comment:
it's a bit verbose - maybe have a special wrapper for it?

Reply:
agree, and fixed.